client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -203,10 +206,41 @@ class StatsPeriodNavigation extends PureComponent {
 			<div
 				className={ clsx( 'stats-period-navigation', {
 					'stats-period-navigation__is-with-new-date-control': isWithNewDateControl,
+					'stats-period-navigation__is-with-new-date-filtering': isWithNewDateFiltering,
 				} ) }
 			>
 				<div className="stats-period-navigation__children">{ children }</div>
-				{ isWithNewDateControl ? (
+				{ isWithNewDateFiltering ? (
```

Comment:
This new condition would cause some `StatsPeriodNavigation` usage without `isWithNewDateControl` and `isWithNewDateFiltering` to show an error when rendering because they only aimed to show `NavigationArrows`, e.g., the [StatsEmailDetail](https://github.com/Automattic/wp-calypso/blob/f246f5ed0dcea667b9954b0853fec1a15fa14a6f/client/my-sites/stats/stats-email-detail/index.jsx#L253-L267) page.

Reply:
Ah, good point. Thanks for flagging this!